project/src/main/java/com/google/sps/Servlets/AuthServlet.java

```diff
@@ -22,12 +22,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/login.jsp"; //change to UserServlet
+      String urlToRedirectToAfterUserLogsOut = "/User"; //change to UserServlet
```

Comment:
also append the hash / user id so the link is unique

Reply:
Unfortunately at this point we have no way or getting the user id because we are generating a link that will redirect the user to the Google log in page. Here I am setting the redirect url to which the user will be routed after they log out. Since we have a session attribute set with the user id we will still know who just signed out and be able to act appropriately